test/e2e/artifact_test.go

```diff
@@ -142,58 +128,208 @@ var _ = Describe("Podman artifact", func() {
 		Expect(err).ToNot(HaveOccurred())
 
 		artifact1Name := fmt.Sprintf("localhost:%s/test/artifact1", port)
-		podmanTest.PodmanExitCleanly([]string{"artifact", "add", artifact1Name, artifact1File}...)
+		podmanTest.PodmanExitCleanly("artifact", "add", artifact1Name, artifact1File)
```

Comment:
just in general, I would expect these unrelated changes as part of another commit. They are great cleanup but it should be in an extra commit to make reviewing much easier.

Reply:
I put the test cleanup in a separate commit.